client/src/app/shared/misc/number-formatter.pipe.ts

```diff
@@ -5,12 +5,15 @@ import { Pipe, PipeTransform } from '@angular/core'
 @Pipe({ name: 'myNumberFormatter' })
 export class NumberFormatterPipe implements PipeTransform {
   private dictionary: Array<{max: number, type: string}> = [
-    { max: 1000, type: '' },
+    { max: 1000, type: 'B' },
```

Comment:
Please remove it because we use this number formatter to count views (and we don't want to have 102B views).

You could add an optional `suffix: string` parameter in the transform method.